index.html

```diff
@@ -15,8 +15,60 @@
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
+  <!-- Body -->
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <main>
+    <h1 class="main-text">CELESTE</h1>
+    <h2>Madeline's adventure</h2>
+    <p class="first-text">One of My Favorites Games Ever</p>
+    
+    <div class="video">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/70d9irlxiB4" title="YouTube video player"
+      frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+      allowfullscreen></iframe>
+      </div>
+      </main>
+      
```

Comment:
I wonder if this could have been wrapped inside a header element, instead of stored within main?